integration-test/1732-restore-building-scale-rank.py

```diff
@@ -85,7 +85,7 @@ def test_merged_z15(self):
             z, x, y, 'buildings', {
                 'kind': 'building',
                 'scale_rank': 5,
-            }, 1)
+            }, 10)
```

Comment:
there should be more changes to fix this, right now the integ test only passes if asserting 10 buildings.